pkg/agent/templates/templates.go

```diff
@@ -34,6 +34,7 @@ const ContainerdConfigTemplate = `
 
 {{- if .NodeConfig.AgentConfig.Snapshotter }}
 [plugins.cri.containerd]
+  disable_snapshot_annotations = true
```

Comment:
where is this coming from?

Reply:
It's covered in the PR description and commit message:
> Disable containerd experimental snapshot labels
> Related to #2455 and containerd/containerd#4684
> These were not meant to be enabled by default, break images with many layers, and will be disabled by default on the next containerd release.